plugins/defaultplugins/ifplugin/vppdump/dump_vppcalls_test.go

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppdump
```

Comment:
vppdump_test

Also the coverage of the dump_vppcalls.go is quite low (<70%). I know there are some functions like 'processIPDetails' which cannot be tested in current implementation, but maybe you could add more scenarios (test more than one interface type, try ipv6, make test expecting errors, etc.)